src/main/java/com/github/susom/database/Database.java

```diff
@@ -200,7 +200,9 @@
 
   /**
    * Convenience method to check if a table or view exists so that caller can decide
-   * whether to create or update a table.  The default catalog and schema will be used.
+   * whether to create or update a table.  The lookup is case insensitive using upper or
+   * lower case depending on the DB in use, unless the tableName is double quoted.
```

Comment:
I have a feeling that this wording will be confusing to people from the future who haven't spent the last week discussing this topic.  Possible alternative: The table name's case is normalized using the database's convention unless tableName is enclosed in double quotes.

Reply:
That works.  Thank you.  Will put it in for the next commit.

Reply:
I also modified some naming because I think "normalizedTableName" is more clear.